packages/destination-actions/src/destinations/stackadapt-audiences/index.ts

```diff
@@ -65,14 +65,13 @@ const destination: DestinationDefinition<Settings> = {
     const userId = payload.userId
     const formattedExternalIds = `["${userId}"]`
 
-    const syncIds = [sha256hash(String(userId))]
-    const formattedSyncIds = `[${syncIds.map((syncId: string) => `"${syncId}"`).join(', ')}]`
+    const syncId = [sha256hash(String(userId))]
 
     const mutation = `mutation {
       deleteProfilesWithExternalIds(
         externalIds: ${formattedExternalIds},
         externalProvider: "${EXTERNAL_PROVIDER}",
-        syncIds: ${formattedSyncIds}
+        syncIds: ${syncId}
```

Comment:
I don't think the brackets are preserved when you convert an array to a string, if you want that

Reply:
They should be called like syncId: "28test", so I adjusted the format to that

Reply:
Okay, but what is the purpose of the syncId variable still being an array?

Reply:
Right, I'll fix it as well